tests/unit_tests/test_point.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
```

Comment:
Please remove the ```as np```. This has been a bad habit in the past, and should not be continued.

Reply:
done